sql/moz-fx-data-shared-prod/search_terms_derived/adm_daily_aggregates_v1/query.sql

```diff
@@ -5,7 +5,7 @@ SELECT
   COUNT(*) AS impressions,
   COUNTIF(is_clicked) AS clicks,
 FROM
-  search_terms_derived.suggest_impression_sanitized_v2
+  `moz-fx-data-shared-prod.search_terms_derived.suggest_impression_sanitized_v2`
```

Comment:
@whd there are a few `search_terms` queries which are getting fully-qualified in this PR that need your approval. (Or we can add them to a skip list if for some reason they shouldn't be fully-qualified)